cmd/field-create/field_create.go

```diff
@@ -145,10 +145,11 @@ func createFieldArgs(config createFieldConfig) (*createProjectV2FieldMutation, m
 	}
 
 	if len(config.opts.singleSelectOptions) != 0 {
-		opts := make([]githubv4.ProjectV2SingleSelectFieldOptionInput, len(config.opts.singleSelectOptions))
+		opts := make([]githubv4.ProjectV2SingleSelectFieldOptionInput, 0)
```

Comment:
this was sending bad data when used with append below, since this will usually be small it's simpler to stick to conventions and not initialize the length even though it's known